### PR TITLE
Present bandit table columns in tabular format

### PR DESCRIPTION
**Pull request by aarsilv**
Coming off a call with Coinbase, they wanted detailed information about the bandit assignment table they need to create in their data warehouse. While we had the required columns in a bulleted-list format, it was less clear (to the point I've been pointing people at a different piece of documentation) than how we describe things like event logs at the SDK level.

The ask was to restructure the required columns in a tabular format and include the expected data type for each.

This PR makes that change, and--like our SDK event documentation--provides an example value as well.

image

See the fully rendered documentation using the autogenerated link, under the "Contextual multi-armed Bandits
" section (`/sdks/sdk-features/bandits/`)

👉 [Direct Link](https://deploy-preview-443--eppo-data-docs.netlify.app/sdks/sdk-features/bandits/#logging-bandit-assignments) 👈

### CODE REVIEW

File: docs/sdks/server-sdks/node.md [hunk 1]
```diff
@@ -413,7 +413,7 @@ The SDK will invoke the `logBanditAction()` function with an `IBanditEvent` obje
 | `subjectNumericAttributes` (Attributes)     | Metadata about numeric attributes of the subject. Map of the name of attributes their provided values             | `{"age": 30}`                  |
 | `subjectCategoricalAttributes` (Attributes) | Metadata about non-numeric attributes of the subject. Map of the name of attributes their provided values         | `{"loyalty_tier": "gold"}`     |
 | `action` (string)                           | The action assigned by the bandit                                                                                 | "promo-20%-off"                |
-| `actionNumericAttributes` (Attributes)      | Metadata about numeric attributes of the assigned action. Map of the name of attributes their provided values     | `{"discount": 0.1}`            |
+| `actionNumericAttributes` (Attributes)      | Metadata about numeric attributes of the assigned action. Map of the name of attributes their provided values     | `{"discount": 0.2}`            |
```
Comment (aarsilv) [inline docs/sdks/server-sdks/node.md:416]: While in here I figure I'd fix this to be consistent with the example action name

**Comment by schmit** [comment]
There's also a list on this page: [bandit-quickstart/](https://docs.geteppo.com/bandit-quickstart/) that we might want to update

File: docs/quick-starts/bandit-quickstart.md [hunk 2]
```diff
@@ -30,19 +30,21 @@ Eppo leverages your existing event logging infrastructure to track experiment as
...
 This logger should write to a table with columns with the following names (they can be in any order):
-* timestamp - Timestamp of the bandit assignment
-* key - The key (unique identifier) of the bandit
-* subject - The unique identifier for the subject being assigned
-* subject_numeric_attributes - Mapping of attribute names to numbers, in JSON format, for the numeric-valued attributes of the subject
-* subject_categorical_attributes - Mapping of attribute names to strings, in JSON format, for the non-numeric-valued attributes of the subject
-* action - The action assigned by the bandit
-* action_numeric_attributes - Mapping of attribute names to numbers, in JSON format, for the numeric-valued attributes of the assigned action
-* action_categorical_attributes - Mapping of attribute names to strings, in JSON format, for the non-numeric-valued attributes of the assigned action
+* **timestamp** - Timestamp of the bandit assignment
```
Comment (aarsilv) [inline docs/quick-starts/bandit-quickstart.md:33]: bolded column names

File: docs/quick-starts/bandit-quickstart.md [hunk 3]
```diff
@@ -30,19 +30,21 @@ Eppo leverages your existing event logging infrastructure to track experiment as
...
 
 Additional information that is provided to the logger that can optionally--but is recommended--be logged includes:
-* experiment - The name of the experiment
-* action_probability - The probability (weight) given to the assigned action at the time of assignment
-* model_version - The current version identifier of the model used to determine action weights
+* **feature_flag** - The key of the feature flag corresponding to the bandit
+* **model_version** - The current version identifier of the model used to determine action weights
+* **action_probability** - The probability (weight) given to the assigned action at the time of assignment
+* **optimality_gap** - The difference between the score of the selected action and the highest-scored action
+* **metadata** - Any additional freeform metadata, in JSON format, such as the version of the SDK
```
Comment (aarsilv) [inline docs/quick-starts/bandit-quickstart.md:47]: Added some new ones

File: docs/sdks/sdk-features/bandits.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
 Contextual multi-armed bandits use reinforcement learning to adjust the frequency it serves an action from a set provided
 to the SDK.
-They use the attributes of the subject and assigned action to form a context for the assignment, and they also have a 
+They use the attributes of the subject and assigned action to form a context for the assignment, and they also have a
 metric (set in the Eppo application) that it is optimizing for.
 
-Every 24 hours, the bandit looks at the previous 30 days for changes in the optimization metric for subjects who were given
-assignments. It uses the assignment context and metric data to build a model for using context to optimally balance 
+At a specified frequency (e.g., every 24 hours), the bandit looks back at a specified number of days (e.g., 30) for changes in the optimization metric for subjects who were given
```
Comment (aarsilv) [inline docs/sdks/sdk-features/bandits.md:12]: These are configurable now 📈 

image

File: docs/sdks/sdk-features/bandits.md [hunk 5]
```diff
@@ -19,120 +19,101 @@ are strings, so bandits are used with string-typed flags. Actions and attributes
...
-))
-
-Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actionsWithAttributes);
+When requesting an assignment from a flag with a bandit, you can use a specialized method that returns the assigned variation and, if the bandit was invoked, the selected action.
+This method is like getting other assignments, but you also provide the set of actions and their attributes that the bandit should consider.
+
+In the Python, the call may look like:
+
+```python
+# Flag that has a bandit variation
```
Comment (aarsilv) [inline docs/sdks/sdk-features/bandits.md:28]: I liked the comments we had in the Java example so used them with Python

File: docs/sdks/sdk-features/bandits.md [hunk 6]
```diff
@@ -19,120 +19,101 @@ are strings, so bandits are used with string-typed flags. Actions and attributes
...
-    } catch (SQLException e) {
-      throw new RuntimeException("Unable to log bandit assignment "+e.getMessage(), e);
-    }
-})
+The name of the table to store bandit assignments can be chosen by you. When setting up the bandit, you will specify the name of the table to use to process its actions.
+However, the columns must have specific names.
+
+Therefore, the logger should write to a table with the following columns (they can be in any order):
+
+| Column                                | Description                                                                                              | Example Data                |
```
Comment (aarsilv) [inline docs/sdks/sdk-features/bandits.md:81]: Table format now! 💪

**Review by schmit — APPROVED** [review]
sweet!

**Pull request merged**